### PR TITLE
add keyword consumer

### PR DESCRIPTION
**Pull request by BelehovEgor**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/728

### CODE REVIEW

File: src/MessageService.Models.Dto/Requests/ParseEntity/AddKeywordRequest.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using LT.DigitalOffice.MessageService.Models.Dto.Enums;
+
+namespace LT.DigitalOffice.MessageService.Models.Dto.Requests.ParseEntity
+{
+    public class AddKeywordRequest
```
Comment (djino97) [inline src/MessageService.Models.Dto/Requests/ParseEntity/AddKeywordRequest.cs:5]: record

File: src/MessageService.Data/KeywordRepository.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                throw new ArgumentNullException(nameof(entity));
+            }
+
+            _provider.ParseEntities.Add(entity);
+            _provider.Save();
+
+            return entity.Id;
+        }
+
+        public bool DoesKeywordExist(string keyword)
```
Comment (djino97) [inline src/MessageService.Data/KeywordRepository.cs:33]: Why write "Does", but not "Are"?

**Review by djino97 — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

File: src/MessageService.Data/KeywordRepository.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/MessageService.Data/KeywordRepository.cs:33]: Exist is verb, and how i know we use do with verbs and be with adjective

File: src/MessageService.Models.Dto/Requests/ParseEntity/AddKeywordRequest.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/MessageService.Models.Dto/Requests/ParseEntity/AddKeywordRequest.cs]: fixed

**Review by borovHokage — APPROVED** [review]

**Pull request merged**